WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteCardAndItem.kt

```diff
@@ -60,7 +60,8 @@ sealed class MySiteCardAndItem(open val type: Type, open val activeQuickStartIte
         PROMOTE_WITH_BLAZE_CARD,
         DASHBOARD_DOMAIN_CARD,
         PAGES_CARD_ERROR,
-        PAGES_CARD
+        PAGES_CARD,
+        ACTIVITY_CARD,
```

Comment:
np: It doesn't break the compiler, but I prefer to not have an extra comma after the last item in the enum class. 🙂

Reply:
Same as above